lib/vagrant-orchestrate/command/init.rb

```diff
@@ -103,24 +103,27 @@ def execute
           return unless argv
 
           if options[:provisioners].include? "puppet"
+            FileUtils.mkdir_p(File.join(@env.cwd, "puppet"))
             if options[:puppet_librarian_puppet]
               contents = TemplateRenderer.render(Orchestrate.source_root.join("templates/puppet/Puppetfile"))
-              write_file "Puppetfile", contents, options
-              FileUtils.mkdir_p(File.join(@env.cwd, "modules"))
-              write_file(File.join(@env.cwd, "modules", ".gitignore"), "*", options)
+              write_file File.join("puppet", "Puppetfile"), contents, options
+              FileUtils.mkdir_p(File.join(@env.cwd, "puppet", "modules"))
+              write_file(File.join(@env.cwd, "puppet", "modules", ".gitignore"), "*", options)
               options[:plugins] << "vagrant-librarian-puppet"
             end
 
             if options[:puppet_hiera]
               contents = TemplateRenderer.render(Orchestrate.source_root.join("templates/puppet/hiera.yaml"))
-              write_file("hiera.yaml", contents, options)
-              FileUtils.mkdir_p(File.join(@env.cwd, "hiera"))
+              write_file(File.join("puppet", "hiera.yaml"), contents, options)
+              FileUtils.mkdir_p(File.join(@env.cwd, "puppet", "hiera"))
```

Comment:
I believe the correct directory structure of hiera is ./puppet/hieradata/ instead of ./puppet/hiera/

https://docs.puppetlabs.com/hiera/1/puppet.html#automatic-parameter-lookup